### DISCUSSION TITLE
Method to switch to OML v1 documentation

### DISCUSSION THREAD

**Discussion by mprather** [opencaesar/platform#25] [General]
The site https://opencaesar.github.io/oml/ now specifically renders v2 specs. 

A lot of users are still using v1 (and will likely do so for another year or two). There does not appear to be a way to switch the documentation to v1.  

Is the v1 site still available? If so, can a link to it be added to the current v2 documentation?

**Comment by melaasar** [comment]
Thanks for pointing this out. I updated the spec site http://www.opencaesar.io/oml/ to link to the previous version (http://www.opencaesar.io/oml/v1) at the top.

**Comment by mprather** [comment]
Thanks.
